RadixWallet/Features/AssetTransferFeature/Components/ChooseAccount/ChooseReceivingAccount+Reducer.swift

```diff
@@ -77,16 +83,18 @@ public struct ChooseReceivingAccount: Sendable, FeatureReducer {
 	}
 
 	public struct Destination: DestinationReducer {
```

Comment:
Hm, so you did not get the "recursive definition" macro error, despite having a Destination, do you know why?

Reply:
Yes, I found out why it isn't working for `GatewaySettings` and `DappDetails`. The macro error occurs due to the presence of an `AlertState` extension
```swift
extension AlertState<GatewaySettings.Destination.Action.RemoveGatewayAlert>
```